apps/shell/tash_security.c

```diff
@@ -77,6 +80,23 @@ void tash_check_security(int fd)
 			continue;
 		}
 
+		#ifdef CONFIG_TASH_REBOOT
+		if (!strncmp((unsigned char *)input, "reboot", 6)) {
+			/*
+			* Invoke the BOARDIOC_RESET board control to reset the board. If
+			* the board_reset() function returns, then it was not possible to
+			* reset the board due to some constraints.
+			*/
+			boardctl(BOARDIOC_RESET, EXIT_SUCCESS);
+
+			/*
+			* boarctl() will not return in this case.  It if does, it means that
+			* there was a problem with the reset operaion.
+			*/
+			return ERROR;
```

Comment:
This function has void for return so that this should be `return;`, not `return ERROR`. or `break`.